src/components/speaker/SpeakerDetails.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 import { css } from "emotion";
-
+import { AgendaTalksListItem } from "../agenda/AgendaTalksListItem";
```

Comment:
I actually only meant for this component to be used for the agenda talks list.

I think repurposing it for now is definitely fine (and the easiest thing to do), but if we have time, we should make a better display for the user profile page. :^)